### PR TITLE
feat: gas station

### PR DESCRIPTION
**Pull request by angelia-yuqi-personal**
1. support eip1559 in frontend
2. gas station when tx follow eip1559
3. advanced gasPrice / gasLimit / nonce

---
This change is [Reviewable](https://reviewable.io/reviews/conflux-chain/helios/1190)

### CODE REVIEW

File: packages/popup/src/components/NumberInput.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ const exponentialCenter = /^\d+(\.\d+)?([Ee]{1})$/
 const exponential = /^\d+(\.\d+)?([Ee][+-]?[\d]+)?$/
 
 const formatInputValue = (targetValue, decimals) => {
+  console.log(!targetValue)
```
Comment (zctocm) [inline packages/popup/src/components/NumberInput.js:12]: remove the console

File: packages/popup/src/pages/AdvancedGas/index.js [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+            amount: 1,
+            unit: 'GDrip',
+          }),
+        )
+      }
+    }
+  }
+
+  const onChangeMaxFeePerGas = maxFeePerGas => {
+    console.log('maxFeePerGas', maxFeePerGas)
```
Comment (zctocm) [inline packages/popup/src/pages/AdvancedGas/index.js:165]: remove the console

File: packages/popup/src/pages/AdvancedGas/index.js [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+    setInputMaxFeePerGas(maxFeePerGas)
+    if (
+      !maxPriorityFeePerGasErr &&
+      new Big(inputMaxPriorityFeePerGas).gt(maxFeePerGas || '0')
+    ) {
+      setMaxPriorityFeePerGasErr(t('maxPriorityFeePerGasHighErr'))
+    } else if (
+      maxPriorityFeePerGasErr === t('maxPriorityFeePerGasHighErr') &&
```
Comment (zctocm) [inline packages/popup/src/pages/AdvancedGas/index.js:173]: I think maybe ` maxPriorityFeePerGasErr === t('maxPriorityFeePerGasHighErr')` is not required

Comment (angelia-yuqi-personal) [reply packages/popup/src/pages/AdvancedGas/index.js:173]: 1. error will always display under max priority fee input
2. display max priority fee input error first, then check maxFee need >= max priority fee
According above, it's reqiured

File: packages/popup/src/components/NumberInput.js (hunk 1 above)
Comment (angelia-yuqi-personal) [reply packages/popup/src/components/NumberInput.js]: done

File: packages/popup/src/pages/AdvancedGas/index.js (hunk 2 above)
Comment (angelia-yuqi-personal) [reply packages/popup/src/pages/AdvancedGas/index.js]: done

**Review by zctocm — APPROVED** [review]

**Pull request merged**